### PR TITLE
Issue #13213: removed //ok noncompilable/naming

### PR DESCRIPTION
**Pull request by MANISH-K-07**
Part of #13213 
removed '//ok' comments from 13 checks under noncompilable/naming directory.

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationInMultiblockStructuresOne.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/*
+CommentsIndentation
+tokens = (default)SINGLE_LINE_COMMENT, BLOCK_COMMENT_BEGIN
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+public class InputCommentsIndentationInMultiblockStructuresOne {
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationInMultiblockStructuresOne.java:10]: I don't see usage of new Inputs.

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationInMultiblockStructuresOne.java (hunk 1 above)
Comment (MANISH-K-07) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationInMultiblockStructuresOne.java]: Extremely sorry for that..
I am actually working on three issues parallely and misplaced the files in a different branch.

**Review by romani — APPROVED** [review]
Ok to merge.

Thanks a lot

**Pull request merged**